book/website/project-design/data-governance/data-gov-ml.md

```diff
@@ -0,0 +1,74 @@
+(pd-dg-ml)=
+
+# Data Governance for the Machine Learning Pipeline
+
+A Machine Learning (ML) pipeline consists of a series of activities including the collection of data, training of an ML model, and the deployment of the model into use. 
+Data is integral throughout the ML process and the methods for which data is collected, annotated, processed, and shared will impact individuals and communities who may be represented in or the creators of the data, as well as data users who would like access to the data. 
+
+This chapter will cover examples of data governance practices for ML for different steps in the pipeline, including: 
+- Data Collection
+- Data Management
+- Data Processing
+
+(pd-dg-ml-dc)=
+## Data Collection
+
+Many ML models are trained using datasets collected by a research team, which may be proprietary, or by using an open dataset that is available for download (sometimes with restrictions on its usage, such as only available for academic researchers).
+The deep learning (DL) family of models, in particular, rely on massive corpuses of data such as text, code, images, sound, and other media.
+The process of data collection depends on the type and volume of data required and sources for acquisition.
```

Comment:
Perhaps a few examples might be good to cite here? I.e. a health project that requires analysing historical health records as opposed to media sentiment analysis on public opinion for elections, which would analyse social media?